Packs/AnsibleTower/Playbooks/Ansible_Tower-_job_generic_polling.yml

```diff
@@ -193,8 +201,7 @@ tasks:
       Timeout:
         simple: "10"
       dt:
-        simple: AnsibleAWX.Job(val.Job.status !== 'success' | val.Job.status !== 'failed'
-          | val.Job.status !== 'error' | val.Job.status !== 'canceled' ).id
+        simple: AnsibleAWX.Job(val.status!=='failed'&&val.status!=='successful').id
```

Comment:
@ayman-m What about "canceled" and "error"?

Reply:
@tomer-pan "canceled" and "error" are not statues as per Ansible's documentation https://docs.ansible.com/ansible-tower/latest/html/userguide/jobs.html

Reply:
Indeed a bit confusing since [in this documentation](https://docs.ansible.com/ansible-tower/3.2.3/html/towerapi/job_list.html) we can definitely see these statuses:
![image](https://user-images.githubusercontent.com/81556849/157006919-be24345c-38ce-4b01-a78e-d9286d520036.png)
I would say that we should handle it on the safe side and add the additional statuses just in case. What do you think?

Reply:
And here is also the latest version api with the same statuses (I posted a non-updated api reference in the last comment):
https://docs.ansible.com/ansible-tower/latest/html/towerapi/api_ref.html#/Instances/Instances_instances_jobs_list
![image](https://user-images.githubusercontent.com/81556849/157007487-66d90870-2e52-4d03-a40a-e37252732632.png)

Reply:
@tomer-pan you are right let's be on the safe side, will push another commit now